app/helpers/application_helper.rb

```diff
@@ -38,6 +38,11 @@ def flash_allowance(type, result, flash_type = flash, most_specific: false)
     result
   end
 
+  def flash_csv_result(result)
```

Comment:
Since this is associated just with CSV processing, a better location for this method is under the `MarkusCsv` class